
 equal to:

   find sys -type f  -exec sed -i {} -e "/^__FBSDID/d" \;
   find lib -type f  -exec sed -i {} -e "/^__FBSDID/d" \;

---
 lib/libcam/camlib.c             |    1 -
 lib/libcam/scsi_cmdparse.c      |    1 -
 lib/libdevstat/devstat.c        |    1 -
 lib/libgeom/geom_util.c         |    1 -
 lib/libipx/ipx_addr.c           |    1 -
 lib/libipx/ipx_ntoa.c           |    1 -
 lib/libkvm/kvm.c                |    1 -
 lib/libkvm/kvm_amd64.c          |    1 -
 lib/libkvm/kvm_arm.c            |    1 -
 lib/libkvm/kvm_cptime.c         |    1 -
 lib/libkvm/kvm_file.c           |    1 -
 lib/libkvm/kvm_getloadavg.c     |    1 -
 lib/libkvm/kvm_getswapinfo.c    |    1 -
 lib/libkvm/kvm_i386.c           |    1 -
 lib/libkvm/kvm_minidump_amd64.c |    1 -
 lib/libkvm/kvm_minidump_i386.c  |    1 -
 lib/libkvm/kvm_pcpu.c           |    1 -
 lib/libkvm/kvm_proc.c           |    1 -
 lib/libkvm/kvm_sparc.c          |    1 -
 lib/libkvm/kvm_sparc64.c        |    1 -
 lib/libusbhid/data.c            |    1 -
 lib/libusbhid/descr.c           |    1 -
 lib/libusbhid/parse.c           |    1 -
 lib/libusbhid/usage.c           |    1 -
 sys/cam/cam.c                   |    1 -
 sys/cam/cam_periph.c            |    1 -
 sys/cam/cam_queue.c             |    1 -
 sys/cam/cam_sim.c               |    1 -
 sys/cam/cam_xpt.c               |    1 -
 sys/cam/scsi/scsi_all.c         |    1 -
 sys/cam/scsi/scsi_cd.c          |    1 -
 sys/cam/scsi/scsi_da.c          |    1 -
 sys/cam/scsi/scsi_low.c         |    1 -
 sys/cam/scsi/scsi_low_pisa.c    |    1 -
 sys/cam/scsi/scsi_pass.c        |    1 -
 sys/cam/scsi/scsi_pt.c          |    1 -
 sys/cam/scsi/scsi_sa.c          |    1 -
 sys/cam/scsi/scsi_ses.c         |    1 -
 sys/cam/scsi/scsi_sg.c          |    1 -
 sys/cam/scsi/scsi_targ_bh.c     |    1 -
 sys/cam/scsi/scsi_target.c      |    1 -
 sys/kern/subr_sbuf.c            |    1 -
 42 files changed, 42 deletions(-)

--- a/lib/libcam/camlib.c
+++ b/lib/libcam/camlib.c
@@ -24,7 +24,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libcam/camlib.c,v 1.16.2.1.4.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/types.h>
 #include <sys/param.h>
--- a/lib/libcam/scsi_cmdparse.c
+++ b/lib/libcam/scsi_cmdparse.c
@@ -35,7 +35,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libcam/scsi_cmdparse.c,v 1.7.34.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/types.h>
 
--- a/lib/libdevstat/devstat.c
+++ b/lib/libdevstat/devstat.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libdevstat/devstat.c,v 1.29.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/types.h>
 #include <sys/sysctl.h>
--- a/lib/libgeom/geom_util.c
+++ b/lib/libgeom/geom_util.c
@@ -25,7 +25,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libgeom/geom_util.c,v 1.1.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/disk.h>
--- a/lib/libipx/ipx_addr.c
+++ b/lib/libipx/ipx_addr.c
@@ -31,7 +31,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libipx/ipx_addr.c,v 1.9.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 static char sccsid[] = "@(#)ipx_addr.c";
--- a/lib/libipx/ipx_ntoa.c
+++ b/lib/libipx/ipx_ntoa.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libipx/ipx_ntoa.c,v 1.7.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 static char sccsid[] = "@(#)ipx_ntoa.c";
--- a/lib/libkvm/kvm.c
+++ b/lib/libkvm/kvm.c
@@ -32,7 +32,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm.c,v 1.31.2.1.4.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libkvm/kvm_amd64.c
+++ b/lib/libkvm/kvm_amd64.c
@@ -32,7 +32,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_amd64.c,v 1.22.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libkvm/kvm_arm.c
+++ b/lib/libkvm/kvm_arm.c
@@ -36,7 +36,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_arm.c,v 1.4.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/elf32.h>
--- a/lib/libkvm/kvm_cptime.c
+++ b/lib/libkvm/kvm_cptime.c
@@ -29,7 +29,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_cptime.c,v 1.1.2.1.4.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/pcpu.h>
--- a/lib/libkvm/kvm_file.c
+++ b/lib/libkvm/kvm_file.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_file.c,v 1.18.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libkvm/kvm_getloadavg.c
+++ b/lib/libkvm/kvm_getloadavg.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_getloadavg.c,v 1.10.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libkvm/kvm_getswapinfo.c
+++ b/lib/libkvm/kvm_getswapinfo.c
@@ -25,7 +25,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_getswapinfo.c,v 1.27.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/time.h>
--- a/lib/libkvm/kvm_i386.c
+++ b/lib/libkvm/kvm_i386.c
@@ -32,7 +32,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_i386.c,v 1.21.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libkvm/kvm_minidump_amd64.c
+++ b/lib/libkvm/kvm_minidump_amd64.c
@@ -24,7 +24,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_minidump_amd64.c,v 1.2.10.1 2009/04/15 03:14:26 kensmith Exp $");
 
 /*
  * AMD64 machine dependent routines for kvm and minidumps. 
--- a/lib/libkvm/kvm_minidump_i386.c
+++ b/lib/libkvm/kvm_minidump_i386.c
@@ -24,7 +24,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_minidump_i386.c,v 1.2.10.1 2009/04/15 03:14:26 kensmith Exp $");
 
 /*
  * AMD64 machine dependent routines for kvm and minidumps. 
--- a/lib/libkvm/kvm_pcpu.c
+++ b/lib/libkvm/kvm_pcpu.c
@@ -29,7 +29,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_pcpu.c,v 1.1.2.1.4.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/pcpu.h>
--- a/lib/libkvm/kvm_proc.c
+++ b/lib/libkvm/kvm_proc.c
@@ -38,7 +38,6 @@
 #endif
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_proc.c,v 1.94.2.1.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 /*
  * Proc traversal interface for kvm.  ps and w are (probably) the exclusive
--- a/lib/libkvm/kvm_sparc.c
+++ b/lib/libkvm/kvm_sparc.c
@@ -32,7 +32,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_sparc.c,v 1.7.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libkvm/kvm_sparc64.c
+++ b/lib/libkvm/kvm_sparc64.c
@@ -34,7 +34,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libkvm/kvm_sparc64.c,v 1.9.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #if defined(LIBC_SCCS) && !defined(lint)
 #if 0
--- a/lib/libusbhid/data.c
+++ b/lib/libusbhid/data.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libusbhid/data.c,v 1.6.32.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <assert.h>
 #include <stdlib.h>
--- a/lib/libusbhid/descr.c
+++ b/lib/libusbhid/descr.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libusbhid/descr.c,v 1.8.32.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/types.h>
 
--- a/lib/libusbhid/parse.c
+++ b/lib/libusbhid/parse.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libusbhid/parse.c,v 1.10.28.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <assert.h>
 #include <stdlib.h>
--- a/lib/libusbhid/usage.c
+++ b/lib/libusbhid/usage.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/lib/libusbhid/usage.c,v 1.8.32.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <assert.h>
 #include <ctype.h>
--- a/sys/cam/cam.c
+++ b/sys/cam/cam.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/cam.c,v 1.11.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #ifdef _KERNEL
--- a/sys/cam/cam_periph.c
+++ b/sys/cam/cam_periph.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/cam_periph.c,v 1.69.2.5.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/systm.h>
--- a/sys/cam/cam_queue.c
+++ b/sys/cam/cam_queue.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/cam_queue.c,v 1.9.20.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/systm.h>
--- a/sys/cam/cam_sim.c
+++ b/sys/cam/cam_sim.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/cam_sim.c,v 1.11.2.2.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/systm.h>
--- a/sys/cam/cam_xpt.c
+++ b/sys/cam/cam_xpt.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/cam_xpt.c,v 1.190.2.19.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/bus.h>
--- a/sys/cam/scsi/scsi_all.c
+++ b/sys/cam/scsi/scsi_all.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_all.c,v 1.51.2.3.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 
--- a/sys/cam/scsi/scsi_cd.c
+++ b/sys/cam/scsi/scsi_cd.c
@@ -46,7 +46,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_cd.c,v 1.102.2.3.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include "opt_cd.h"
 
--- a/sys/cam/scsi/scsi_da.c
+++ b/sys/cam/scsi/scsi_da.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_da.c,v 1.224.2.6.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 
--- a/sys/cam/scsi/scsi_low.c
+++ b/sys/cam/scsi/scsi_low.c
@@ -2,7 +2,6 @@
 /*	$NetBSD$	*/
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_low.c,v 1.29.2.1.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #define	SCSI_LOW_STATICS
 #define	SCSI_LOW_DEBUG
--- a/sys/cam/scsi/scsi_low_pisa.c
+++ b/sys/cam/scsi/scsi_low_pisa.c
@@ -32,7 +32,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_low_pisa.c,v 1.8.20.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #ifdef	__NetBSD__
 #include <sys/param.h>
--- a/sys/cam/scsi/scsi_pass.c
+++ b/sys/cam/scsi/scsi_pass.c
@@ -26,7 +26,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_pass.c,v 1.48.2.2.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/systm.h>
--- a/sys/cam/scsi/scsi_pt.c
+++ b/sys/cam/scsi/scsi_pt.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_pt.c,v 1.47.2.1.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/queue.h>
--- a/sys/cam/scsi/scsi_sa.c
+++ b/sys/cam/scsi/scsi_sa.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_sa.c,v 1.113.2.1.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/queue.h>
--- a/sys/cam/scsi/scsi_ses.c
+++ b/sys/cam/scsi/scsi_ses.c
@@ -25,7 +25,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_ses.c,v 1.35.2.2.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/queue.h>
--- a/sys/cam/scsi/scsi_sg.c
+++ b/sys/cam/scsi/scsi_sg.c
@@ -30,7 +30,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_sg.c,v 1.9.2.3.2.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/systm.h>
--- a/sys/cam/scsi/scsi_targ_bh.c
+++ b/sys/cam/scsi/scsi_targ_bh.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_targ_bh.c,v 1.25.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 #include <sys/queue.h>
--- a/sys/cam/scsi/scsi_target.c
+++ b/sys/cam/scsi/scsi_target.c
@@ -28,7 +28,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/cam/scsi/scsi_target.c,v 1.73.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 
 #include <sys/param.h>
--- a/sys/kern/subr_sbuf.c
+++ b/sys/kern/subr_sbuf.c
@@ -27,7 +27,6 @@
  */
 
 #include <sys/cdefs.h>
-__FBSDID("$FreeBSD: src/sys/kern/subr_sbuf.c,v 1.30.8.1 2009/04/15 03:14:26 kensmith Exp $");
 
 #include <sys/param.h>
 
